lib/rucio/core/did_meta_plugins/filter_engine.py

```diff
@@ -344,13 +354,24 @@ def create_sqla_query(self, session=None, additional_model_attributes=[], additi
                         expression = or_(expression, key.is_(None))
                 elif json_column:                                                                   # -> this key filters on the content of a json column
                     if session.bind.dialect.name == 'oracle':
-                        # Compiling for Oracle yields a UnsupportedCompilationError so fall back to support for equality only.
-                        # TODO
-                        if oper != operator.eq:
-                            raise exception.FilterEngineGenericError("Oracle implementation does not support this operator.")
-                        if isinstance(value, str) and any([char in value for char in ['*', '%']]):
-                            raise exception.FilterEngineGenericError("Oracle implementation does not support wildcards.")
-                        expression = text("json_exists(meta,'$?(@.{} == \"{}\")')".format(key, value))
+                        if isinstance(value, str) and any([char in value for char in ['*', '%']]):  # wildcards
+                            if value in ('*', '%', u'*', u'%'):                                     # match wildcard exactly == no filtering on key
+                                continue
+                            else:                                                                   # partial match with wildcard == like || notlike
+                                if oper == operator.eq:
+                                    expression = text("json_exists({},'$?(@.{} like \"{}\")')".format(json_column.key, key, value.replace('*', '%')))
```

Comment:
i'm just going to assume this Oracle syntax does what I think it's doing... 😅

Reply:
holy moly! :-D

Reply:
Yeah :p the tests look fine, and I also tested it in our ESCAPE instance.
One thing maybe to consider, is the differences of `like`, `like_regex` and `eq_regex` as described in [1].
I think the `like` expression matches best what we want in this case, but please feel free to take a look and let me know if you think some of the others would work better (based on their description).